__tests__/app.test.js

```diff
@@ -0,0 +1,74 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+const testData = require("../db/data/test-data/index.js");
+
+// before each test is run, use SEED to re-seed the data so we start
+// when testing, this is the ONLY way we actually seed the test database
+// i.e. developement database usesRUN
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("Status 200: the topics endpoint exists and responds", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
```

Comment:
This test is redundant as the next tests will not pass unless status code is 200.